src/_C017.cpp

```diff
@@ -125,7 +125,7 @@ bool do_process_c017_delay_queue(int controller_number, const Queue_element_base
         continue;                                    // Zabbix will ignore an empty key anyway
       }
       JsonObject block = data.createNestedObject();
-      block[F("host")] = Settings.Name;              // Zabbix hostname, Unit Name for the ESP easy
+      block[F("host")] = Settings.getUnitname();     // Zabbix hostname, Unit Name for the ESP easy
```

Comment:
This may break existing setups (if any, as some users reported Zabbix controller seems broken)